docs/privacy.md

```diff
@@ -75,9 +75,7 @@ We may use the data collected through cookies, log file, device identifiers, loc
 
 We use third-party automated devices and applications, such as Google Analytics, to evaluate usage of our Website. Google Analytics is a Web-based analysis tool provided by the Google Inc., 1600 Amphitheatre Pkwy, Mountain View, CA 94043-1351, USA. Google provides further information about its own privacy practices and offers a browser add-on to opt out of Google Analytics tracking at <a href="https://tools.google.com/dlpage/gaoptout">https://tools.google.com/dlpage/gaoptout</a>.
```

Comment:
Do you need to update "Last modified on" in this file? (On line 7, can't comment on that line in this diff)

Reply:
Good catch and has been updated.